crates/viewer/re_time_panel/src/lib.rs

```diff
@@ -751,35 +751,32 @@ impl TimePanel {
             );
         }
 
+        let store = entity_db.store().read();
```

Comment:
Note about read coarseness here.

Also these show_ methods should probably take handles as parameters... but we cannot patch everything in the world in this PR. Maybe a TODO.